### DISCUSSION TITLE
[M-Z] (10 Febbraio 2020) Esercizio 3

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#460] [Esami & Esoneri M-Z]
## Traccia

<img width="842" alt="image" src="https://github.com/user-attachments/assets/09e2804b-d5a4-4190-b2d7-3fb99954ea79">

**Comment by rimaout** [comment]
<img width="800" alt="image" src="https://github.com/user-attachments/assets/b45f542e-5106-4431-be2c-e9bccad9ce39">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200210.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200210.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/0620c35e-a5ae-4e0b-86c5-8cbd876c2a0a)
